python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/ethernet-interfaces.j2

```diff
@@ -128,10 +170,9 @@
 | Interface | PVLAN Mapping | Secondary Trunk |
 | --------- | ------------- | ----------------|
 {%         for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort('name') %}
-{%             if ethernet_interface.pvlan_mapping is arista.avd.defined or
-               ethernet_interface.trunk_private_vlan_secondary is arista.avd.defined %}
-{%                 set row_pvlan_mapping = ethernet_interface.pvlan_mapping | arista.avd.default('-') %}
-{%                 set row_trunk_private_vlan_secondary = ethernet_interface.trunk_private_vlan_secondary | arista.avd.default('-') %}
+{%             set row_pvlan_mapping = ethernet_interface.switchport.pvlan_mapping | arista.avd.default(ethernet_interface.pvlan_mapping, '-') %}
```

Comment:
i know the logic works but for code consistent lets check first if defined

Reply:
The default filter contains the check for defined. That is the whole purpose of that filter, so let's not repeat extra checks.